src/main/resources/com/owlcyberdefense/ntp/xsd/types.dfdl.xsd

```diff
@@ -57,11 +57,39 @@
   -->
 
   <complexType name="ntpDate">
-    <sequence>
-      <xs:element name="eraNumber" type="xs:int" dfdl:length="32"/>
-      <xs:element name="eraOffset" type="xs:int" dfdl:length="32"/>
-      <xs:element name="fraction" type="xs:int" dfdl:length="64"/>
-    </sequence>
+	  <sequence>
+		  <xs:element name="eraNumber" type="xs:int" dfdl:length="32"/>
+		  <xs:element name="eraOffset" type="xs:int" dfdl:length="32"/>
+		  <xs:element name="fraction" type="xs:int" dfdl:length="64"/>
+	  </sequence>
   </complexType>
 
+  <simpleType name="LI" dfdl:length="2">
+	  <restriction base="xs:unsignedByte"/>
```

Comment:
To fix exlicit vs. implicit length kind issue. Define our own simple types "sint32" and "uint32" and derive all the numbers we use from those. That way only those have lengthKind 'explicit', everything else has length kind 'implicit' by default. (Do in a subsequent PR).